corehq/apps/app_manager/tests/test_form_workflow.py

```diff
@@ -378,14 +378,11 @@ def test_form_links_other_child_module(self):
         """
         self.assertXmlPartialEqual(expected, factory.app.create_suite(), "./entry[2]/stack")
 
-        # m2f0 links to m1f0, a form in a separate child module, here's the stack there
```

Comment:
I'd leave in this first line to explain why there are two assert statements.  There's also a comment at the top of the method saying that the demonstrated behavior is flawed.  Well not anymore!

Also, I just realized I called `factory.app.create_suite()` twice in a row :grimacing: